src/cmd_vel_mux.cpp

```diff
@@ -30,37 +30,62 @@
 
 namespace cmd_vel_mux
 {
-/*****************************************************************************
- ** Implementation
- *****************************************************************************/
 
-CmdVelMux::CmdVelMux(rclcpp::NodeOptions options) : rclcpp::Node("cmd_vel_mux", options.allow_undeclared_parameters(true).automatically_declare_parameters_from_overrides(true)), allowed_(VACANT)
+namespace
 {
-  std::vector<std::string> names;
-  if (!get_parameter("subscribers.name", names))
+std::vector<std::string> stringSplit(const std::string &str,
```

Comment:
@Lobotuerk @clalancette consider using `rcpputils::split` instead.

Reply:
I didn't know about that one.  Good call, fixed in 6091ab8532173b77448379318c2d7c1bb47dec2e